src/content/docs/style-guide/images/create-videos.mdx

```diff
@@ -0,0 +1,24 @@
+---
+title: Create videos
+tags:
+  - Tech writer style guide
+  - Processes and procedures
+redirects:
+---
+
+There are many ways to create videos, but here are some tips that will likely apply to many video production projects: 
+
+* Before you start, talk to the Nerd Bytes team and make sure they're not working on a similar video. 
+* For the most part, try to keep videos under two minutes. If your script is longer than that, consider breaking it up into more than one video. 
+* You can use Camtasia to record your screen capture and voice-over audio. Tips for this: 
```

Comment:
Maybe split out the Camtasia-specific steps into their own section? Or break this list of bullets up into some other (2-3) categories?

Maybe the categories:
Set up
Recording
Editing